collections/_blogs/2020-07-16-community-bridge-with-layer5.md

```diff
@@ -0,0 +1,56 @@
+
+However, at the moment, no service meshes adhere to the SMI specifications completely. Service meshes are rapidly developing and making active efforts to extend their feature set. Simultaneously, SMI has also been working towards adding more features to its specifications.
+<br>
+Consequently, it becomes a necessity for continual validation of service mesh conformance with respect to SMI specifications and  the various versions of those specifications. The process of verifying conformance needs to be standardized and tooling made available.  Hence, our project efforts to create an SMI conformance tool.
+
+<h4> Meshery to the rescue </h4>
+
+Meshery is the multi-service mesh management plane, offering lifecycle, configuration, and performance management of service meshes and their workloads. It is closely connected to the Kubernetes cluster and allows easy deployment of various service meshes and sample applications onto those meshes. Meshery enables operators to run performance tests on workloads, collect, store, and manage the test results.
+<br>
+We had found our solution. By leveraging the features provided in Meshery, we would be able to focus on the conformance validation and testing instead of sweating over the complexity in deployments. We can also use the performance management infrastructure to manage the conformance tests and their results. And so,  the tool will be developed as an integral part of Meshery to aid and abate SMI conformance woes.
+We are now enhancing Meshery to automate the process of verifying conformance against a standard set of tests against a sample workload designed specifically for SMI conformance testing. By the time we’re through, this capability of Meshery will allow each service mesh project to validate its conformance to each version of the SMI specifications and will provide a detailed report.
+
+<h4> My Projects </h4>
+<div class="left" style="color:gray;position:relative;top:-10px;font-size:1.20em;"><i>Teaching service meshes to be compliant</i></div><br>
+
+The goals accomplished in my tenure at Layer5 are summarized as follows:
+<br>
+* Ported meshery and its remote provider from using session based authentication to JWT based auth. This opened doors to:
+
+1. Usage of refresh tokens for longevity of sessions instead of asking user to re-login frequently.
+2. CLI based authentication (not done yet, but is on the roadmap)
+
+* WASM based envoy network packet filters in Rust-lang. Built a demo app demonstrating its potentials; we also built a  WASM filter which performs custom user specific rate limiting. This app was demo’ed in Dockercon 2020 under the header “Service Meshing with Docker Desktop and WebAssembly”.
+* Enhanced meshery’s performance testing and profile management capabilities. Added support for customizing the requests made by the load generator, and added the capability of using performance test profiles in the tool.
+
+
+<h4> Shout out to the Community </h4>
```

Comment:
Careful not to use the same heading name on each of the contributor blog posts. We do want contributor posts to invite people to join the community, but it shouldn't look mechanical if someone reads more than one post.